optimum/exporters/openvino/__main__.py

```diff
@@ -327,7 +320,7 @@ class StoreAttr(object):
         **kwargs_shapes,
     )
 
-    if convert_tokenizer:
+    if convert_tokenizer and is_openvino_tokenizers_available():
```

Comment:
The tokenizer export results in additional files being included which could be confusing for users, can this be moved to to an `openvino_tokenizers` directory ? 

```suggestion
    if convert_tokenizer and is_openvino_tokenizers_available():
         output = Path(output) / "openvino_tokenizers"
```

Reply:
This is similar to the original tokenizer, so we replicate an existing pattern here.

Reply:
I'm thinking about something similar than what's done for [SD models](https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0/tree/main/tokenizer) is this something that sounds reasonable to you? I'm fine with adding it in a following PR if you prefer not including it in this one

Reply:
Yes, I can do that in a separate PR.